jobs/mysql/templates/my.cnf.erb

```diff
@@ -201,6 +201,47 @@ ssl_cert                        = /var/vcap/jobs/mysql/certificates/server-cert.
 ssl_key                         = /var/vcap/jobs/mysql/certificates/server-key.pem
 <% end %>
 
+<% if p('cf_mysql.mysql.performance_schema_enabled') %>
```

Comment:
It looks like most of the values specified are the default values set by MariaDB. We should let MariaDB set the value instead of specifying them in the `my.cnf`, unless there is a specific reason to override the default.

Can you please remove the entries which are setting the default value.

Reply:
Hello @APShirley ,
I set these values to minimize memory consumption by Performance_schema.

I need to change 1 parameter : 
`performance_schema_max_statement_classes               = 178`
because under 178, no statement

Reply:
Hi @ldangeard-orange,
According to the mariadb docs the `performance_schema_max_statement_classes ` ["Should be left as either autosized or disabled, as changing to any positive value has no benefit and will most likely allocate unnecessary memory."](https://mariadb.com/kb/en/library/performance-schema-system-variables/#performance_schema_max_statement_classes). I don't think we want to introduce a change that is not recommended.

Reply:
Hi @APShirley ,
I close this PR, and make a new one without parameter.